src/value/createValueMockFactory.ts

```diff
@@ -34,6 +34,7 @@ import { sliceZone, MockSliceZoneValueConfig } from "./sliceZone";
 import { timestamp, MockTimestampValueConfig } from "./timestamp";
 import { title, MockTitleValueConfig } from "./title";
 import { uid, MockUIDValueConfig } from "./uid";
+import { MockTableValueConfig, table } from "./table";
```

Comment:
The rest of these imports are sorted alphabetically, so I would move this further up. And the rest also switch the order of the imported items, so I would reverse those as well.

Reply:
If we really care about this, let's automate it in a different PR with a formatter/linter plugin or something.

Reply:
Yeah, it's not a big deal.